mobile/src/libs/enums/ui/app-color.enum.ts

```diff
@@ -1,12 +1,17 @@
 const AppColor = {
-  BLUE_100: '#4E80FF',
-  BLUE_200: '#507CEB',
-  PINK_100: '#E1A7D5',
-  RED_100: '#D51A52',
+  BLUE_100: '#E5F0FF',
+  BLUE_200: '#4E80FF',
+  BLUE_300: '#507CEB',
   GRAY_100: '#EBF3FF',
   GRAY_200: '#CCDDF3',
   GRAY_300: '#ACB4D0',
-  GRAY_400: '#302E36',
+  GRAY_400: '#EDF2F7',
+  GRAY_500: '#302E36',
+  PINK_100: '#E1A7D5',
+  RED_100: '#D51A52',
+  TRANSPARENT_BLUE_100: '#051E4703',
```

Comment:
Name this color in the following format:
BLUE_100_ALPHA_(value)

Reply:
Done